### PR TITLE
Fix build issues with clang 7.x

### PR DESCRIPTION
**Pull request by albaguirre**

### CODE REVIEW

File: src/client/cmd/list.cpp [hunk 1]
```diff
@@ -41,13 +41,13 @@ std::string status_string(const multipass::ListVMInstance_Status& status)
...
-    case mp::InfoReply::RUNNING:
+    case mp::ListVMInstance_Status_RUNNING:
         status_val = "RUNNING";
         break;
-    case mp::InfoReply::STOPPED:
+    case mp::ListVMInstance_Status_STOPPED:
         status_val = "STOPPED";
         break;
-    case mp::InfoReply::TRASHED:
+    case mp::ListVMInstance_Status_TRASHED:
```
Comment (townsend2010) [inline src/client/cmd/list.cpp]: I'm not sure I understand why these changes fix clang build issues.

Comment (albaguirre) [reply src/client/cmd/list.cpp]: The error spew:

 error: comparison of two values with different enumeration types in switch statement ('multipass::ListVMInstance_Status' and 'const multipass::InfoReply::Status' (aka 'const multipass::InfoReply_Status'))

Comment (townsend2010) [reply src/client/cmd/list.cpp]: An, ok, cool:)
